cypress/e2e/brewery/ScenarioParametersValidation.cy.js

```diff
@@ -28,6 +28,7 @@ describe('scenario parameters inputs validation', () => {
       GET_WORKSPACES: true,
       GET_ORGANIZATION: true,
       UPDATE_SCENARIO: true,
+      // PERMISSIONS_MAPPING: true,
```

Comment:
Is there a reason to remove the stubbing of the permissions endpoint in this test? (and to enable it in the test ScenarioMetadataParameters?)
Is it related to missing information in the mapping sent by the Cosmo API?

Reply:
It is related to hours when I launched the test :rofl: I launched it when server wasn't running yet, so I added the missing stubbing endpoint to be fully independent from the API, then I disabled it to check that it changes nothing. I will remove it to keep our tests as it was :slightly_smiling_face:

Reply:
Ok, we can keep it then if it helps running our stubbed tests when the API is off :slightly_smiling_face: